index.md

```diff
@@ -5,8 +5,12 @@ title: Hello, We Are Convivio
 
 Convivio is a boutique agency designing and developing digital services for innovative public and private sector organisations.
 
-That means we take clients beyond simply marketing themselves on the web, to delivering their core services online.
+That means we take clients beyond simply marketing themselves on the web, to delivering their core services online, using modern technology and processes to achieve great <br class="only-full-width" />user experiences.
 
-Our team has delivered this digital transformation before in the government, finance, and media sectors.
+We have delivered hugely successfully digital transformation before in the government, finance, and media sectors.
 
-This is release 1 of our website, our ‘minimum launchable product’. We're going to work in the open to develop our site, so you can see our workings-out. Follow thinking and progress on our <a href="https://blog.weareconvivio.com" target="_blank">blog.</a>
+This is release 1 of our website, our ‘minimum launchable product’. We are going to work in the open to develop our site, so you can see our workings-out and our mistakes.
+
+We are very excited on what the future will bring for Convivio. Follow us and join us <br class="only-full-width" />in our journey!
```

Comment:
typo 'excited on' should be 'excited about'
Also, that part of this sentence feels unnecessary